Assignment2/arcecosta-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,67 @@
+{
+	"@context":{
+		"xsd":"http://www.w3.org/2001/XMLSchema#,
+		"s": "http://www.ontologies.org/ontologies/",
+		"prop": "http://www.ontologies.org/properties/",		
+		"includes": "prop:includes",
+		"hasMeasurement": "prop:measurements",
+		"hasOwner": "prop:hasOwner",
+		"hasName": "prop:hasName",
+		"hasTemperature": "prop:hasTemperature",
+		"atTime":"prop:atTime"
+		},
+
+		"Class01":{
+         	"@id":"http://www.ontologies.org/ontologies",
+			"@type":"s:Class01",
+			"includes":[
+				{
+					"Sensor029":{
+						"@id":"http://www.ontologies.org/ontologies/Class01"
+						"@type":"s:Sensor029"
+						"hasMeasurement":
+						{
+							"Measurement8401":
+								{
+									"@id":"http://www.ontologies.org/ontologies/Class01/Sensor029"
+									"@type":"s:Measurement8401"
+									"hasTemperature":
+										{
+											"@value": "29",
+											"@type": "xsd:integer"
+										},
+									"atTime":
+										{
+										"@value": "2010-06-12T12:00:12",
+										"@type": "xsd:integer"
+										}
+								}
+						}
+
+					},
+					{
+					"Computer101":
+					{
+						"@id":"http://www.ontologies.org/ontologies/Class01"
+						"@type":"s:Computer101"				
+						"hasOwner":
+						{
+							"User10A":
+								{
+									"@id":"http://www.ontologies.org/ontologies/Class01/Computer101"
+									"@type":"s:User10A"
+
+									"hasName":
+										{
+										"@value":"Pedro",
+										"@type":"xsd:string"
+										}
+								}
+						}
+
+					}	
+				}
+			}
+			]
+		}
+}
```

Comment:
Please, validate your file before resubmitting it. There are syntax errors.

Reply:
Also, remember that each resource must have an unique identifier. Thus, check the "@id" values.